### PR TITLE
Add -DUSE_BTF=ON cmake configuration parameter on CI

### PR DESCRIPTION
**Pull request by BulatSaif**
[DOPS-292](https://soramitsu.atlassian.net/browse/DOPS-292)

### Description of the Change
add -DUSE_BTF=ON for cmake configuration on CI

### Benefits

It always` false`,  `true` if  build type "Before merge to trunk"

### CODE REVIEW

File: Jenkinsfile-new [hunk 1]
```diff
@@ -152,6 +152,7 @@ node ('master') {
   packagePush = false
   specialBranch = false
   parallelism = 0
+  useBTF = false
 
   if (scmVars.GIT_LOCAL_BRANCH in ["master","develop","dev"] || scmVars.CHANGE_BRANCH_LOCAL in ["develop","dev"])
```
Comment (bakhtin) [inline Jenkinsfile-new]: Shall we get rid of 'dev' branch mentions since we moved back to good old `develop` ?

**Comment by BulatSaif** [comment]
Currently  build fails due to error :
```
CMake Error at /usr/local/Cellar/cmake/3.11.4/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):

  Could NOT find PythonInterp: Found unsuitable version "1.4", but required
  is at least "3.5" (found /Users/jenkins/.pyenv/shims/python3.5)
```
But looks like it work fine on `mac-mcst-1`,
@bakhtin, can you please add pysha3 to rest of mac

**Comment by BulatSaif** [comment]
@bakhtin `mac-mcst-1 `and `mac-mcst-2` works fine, but problem with `mac-mcst-3`

**Comment by BulatSaif** [comment]
mac-mcst-3 now works

**Pull request merged**